config.test.yaml

```diff
@@ -111,6 +111,10 @@ templates:
                   get:
                     backend_request:
                       uri: http://graphoid.wikimedia.org/en.wikipedia.org/v1/png/{title}/{revision}/{graph_id}
+                      headers:
```

Comment:
@gwicke @d00rman These headers were white-listed in the simple_service implementation, so I thought these might be needed. I'll remove them as you say these are unnecessary.